modules/docs/arrow-docs/docs/docs/arrow-core/patterns/errorhandling/README.md

```diff
@@ -168,48 +168,48 @@ How would our example look when implemented with `Try`?
 
 ```kotlin:ank
 
-fun arm(): Try<Nuke> =
-  Try { throw RuntimeException("SystemOffline") }
+fun takeFoodFromRefrigerator(): Try<Lettuce> = 
+    Try { throw RuntimeException("You do not have any lettuce")}
 
-fun aim(): Try<Target> =
-  Try { throw RuntimeException("RotationNeedsOil") }
+fun getKnife(): Try<Knife> = 
+    Try { throw RuntimeException("Your knife is dull") }
 
-fun launch(target: Target, nuke: Nuke): Try<Impacted> =
-  Try { throw RuntimeException("MissedByMeters") }
+fun prepare(knife: Knife, lettuce: Lettuce): Try<Salad> = 
+    Try { throw RuntimeException("You did not have enough lettuce for the recipe") }
 ```
 
 As you can see by the examples below, exceptions are now controlled and caught inside of a `Try`.
 
 ```kotlin:ank
-arm()
+takeFoodFromRefrigerator()
 ```
 
 ```kotlin:ank
-aim()
+getKnife()
 ```
 
 Unlike in the `Option` example here, we can fold over the resulting value accessing the runtime exception.
 
 ```kotlin:ank
-val result = arm()
-result.fold({ ex -> "BOOM!: $ex"}, { "Got: $it" })
+val result = takeFoodFromRefrigerator()
+result.fold({ ex -> "Oh no, I was hungry!: $ex"}, { "Yum: $it" })
 ```
 
 Just like it does for `Option`, Arrow also provides `Monad` instances for `Try`, and we can use it exactly in the same way
 
-```kotlin
+```kotlin:ank
 import arrow.typeclasses.*
 import arrow.core.extensions.*
 
-fun attackTry(): Try<Impacted> =
+fun prepareTry(): Try<Salad> =
   fx.monad {
```

Comment:
```suggestion
  fx {
```
Ank fails with: `error: function invocation 'fx(...)' expected`. So why not try it ^^ If this fails you may need to check the imports again

Reply:
I could try it, but the documentation this code comes with is specifically highlighting the `fx.monad` functionality, so the documentation would become wrong if I did that.

Reply:
It turns out the issue is that I added `:ank` to check this code. The original did not have it. If you put `:ank` around the code before my commit, it errors out.

I'm going to remove the :ank markup to my code and update.

Reply:
`fx {...}` is an alias for `monad().fx.monad {...}` which most datatypes provide. If you want to keep it replace it with `Try.monad().fx.monad {...}`. But you might need to add an import for `monad()` then.